kyo-scheduler/jvm/src/main/scala/kyo/scheduler/Scheduler.scala

```diff
@@ -37,12 +37,6 @@ final class Scheduler(
     @volatile private var allocatedWorkers = 0
     @volatile private var currentWorkers   = coreWorkers
 
-    val a1, a2, a3, a4, a5, a6, a7 = 0L // padding
-
-    @volatile private var cycles = 0L
```

Comment:
No `cycles` anymore since preemption is now done based on `clock.currentMillis`.